lib/src/message/message_builder.dart

```diff
@@ -110,16 +113,26 @@ class MessageMaterial extends StatelessWidget {
 class MessageText extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
+    final markdown = _getText(context).markdown();
+
     return Padding(
       padding: getNamePaddingForGroups(context),
-      child: Text(
-        _getText(context),
-        style: Theme.of(context).textTheme.subhead.apply(color: MessageData.of(context).textColor),
+      child: MarkdownBody(
+        data: markdown,
+        onTapLink: (url) {
+          _launch(url: url);
+        },
       ),
     );
   }
 }
 
+Future<void> _launch({@required String url}) async {
```

Comment:
:+1: we should apply this logic + some sort of error toast / snackbar everywhere. I will create a ticket for that.

Reply:
Which logic do you mean? What is so special in this case...? 🤔

Reply:
Checking for `canLaunch`, not that special, but I know some places where we just call launch without checking ;)